### PR TITLE
Awesome Books

### PR DESCRIPTION
**Pull request by Yordinia**
## In this branch, we have

- Created a collection that ```keeps a list of books.```
 
- Created a function to ```add a new book to the collection, with title and author.```

- Display all books saved in the collection in the top part of the page.

> When a user clicks the "Add" button:

```A new book is added to the collection.
The new book is displayed in the page.`
When a user clicks the "Remove" button
```

>   The correct book is removed from the collection.
>   The right book disappears from the page.
> Data is preserved in the browser's memory by using localStorage.

                 Thank You.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+name: Linters
```
Comment (bellom) [inline .github/workflows/linters.yml:1]: This is not the correct linter config for the JS project, perhaps that is why you got that error. check [this](https://github.com/microverseinc/linters-config/blob/master/javascript/.github/workflows/linters.yml)

**Review by bellom — APPROVED** [review]
@Yordinia,

Make sure to always use the correct linter config.

Because this is the only reason, I will approve this PR

File: .github/workflows/linters.yml (hunk 1 above)
Comment (Yordinia) [reply .github/workflows/linters.yml:1]: Okey thank you. here's the workflow free pull request https://github.com/Yordinia/AwesomeBooks/pull/1

**Comment by bellom** [comment]
@Yordinia  please go ahead and merge

**Pull request merged**